jina/parsers/peapods/runtimes/remote.py

```diff
@@ -5,27 +5,80 @@
 from ....enums import CompressAlgo
 
 
-def mixin_remote_parser(parser):
+def mixin_remote_runtime_parser(parser):
     """Add the options for remote expose
     :param parser: the parser
     """
-    gp = add_arg_group(parser, title='Expose')
+    gp = add_arg_group(parser, title='RemoteRuntime')
+    _add_host(gp)
 
     gp.add_argument(
-        '--host',
-        type=str,
-        default=__default_host__,
-        help=f'The host address of the runtime, by default it is {__default_host__}.',
+        '--port-jinad',
+        type=int,
+        default=12345,  # helper.random_port(),
+        help='The port of the remote machine for usage with JinaD.',
     )
 
+
+def mixin_remote_jinad_parser(parser):
```

Comment:
If this is only used in `daemon` codebase, let's move this to `daemon/parser.py` and set the default `port` to `8000`